client/layout/guided-tours/config-elements.js

```diff
@@ -63,20 +70,36 @@ export class Tour extends Component {
 
 	setTourMeta( props ) {
 		const { branching, next, quit, isValid, name, version, stepName } = props;
-		this.tourMeta = { branching, next, quit, isValid, tour: name, tourVersion: version, step: stepName };
+		this.tourMeta = {
+			branching,
+			next,
+			quit,
+			isValid,
+			tour: name,
+			tourVersion: version,
+			step: stepName,
+			isLastStep: this.isLastStep( { step: stepName, branching } ),
+		};
 	}
 
-	render() {
+	findNextStep() {
 		const { children, stepName } = this.props;
-		const nextStep = find( children, stepComponent =>
+		return find( children, stepComponent =>
```

Comment:
I think it would improve readability if we update the syntax here to use proper bracing, e.g.:

```
return find( ( children, stepComponent ) => {
     stepComponent.props.name === stepName 
} );
```